public/index.html

```diff
@@ -33,6 +33,6 @@ <h1 class="header__logo logo">Cinemaddict</h1>
     <!-- Количество фильмов -->
   </section>
 </footer>
-
+<script src="/bundle.js"></script>
```

Comment:
```suggestion
<script src="./bundle.js"></script>
```
Это чтобы в опубликованной версии пути правильно определялись.
Выше ещё подключение стилей есть, сделай для них также, пожалуйста.